marlowe-dashboard-client/src/Play/State.purs

```diff
@@ -92,6 +91,11 @@ handleAction (ToggleCard card) = do
       | currentCard == card -> handleAction $ SetCard Nothing
     _ -> handleAction $ SetCard $ Just card
 
+handleAction (SetCurrentSlot currentSlot) = do
+  toContractHome $ ContractHome.handleAction $ ContractHome.AdvanceTimeoutedContracts currentSlot
+  modify_
+    $ set (_playState <<< _currentSlot) currentSlot
+
```

Comment:
I don't think we'll need this. It'll be a `handleQuery` case, with the slot coming from the websocket. I've got this working on my branch (that I haven't been able to merge yet because it depends on the new PAB being merged). But we can leave it for now - there'll be some fun merge conflicts to sort out next week... 😬